uefi-test-runner/src/proto/pci/root_bridge.rs

```diff
@@ -53,8 +52,11 @@ pub fn test() {
 
             let (bus, dev, fun) = (addr.bus, addr.dev, addr.fun);
             log::info!(
-                "PCI Device: [{bus}, {dev}, {fun}]: vendor={vendor_id:04X}, device={device_id:04X}, class={class_code:02X}, subclass={subclass_code:02X}"
+                "PCI Device: [{bus:02x}, {dev:02x}, {fun:02x}]: vendor={vendor_id:04X}, device={device_id:04X}, class={class_code:02X}, subclass={subclass_code:02X}"
             );
+            for child_bus in pci_tree.iter_subsequent_busses_for(addr) {
```

Comment:
instead of just iterating, would it make sense to explictely check for the expected PCIe devices?

Reply:
The bus numbers are dynamically allocated and thus **might** vary.
But I've done something like you suggest in the follow-up MR here, based on device paths: https://github.com/rust-osdev/uefi-rs/pull/1831/commits/582714a47c15b774f9a23616be7215a0e5477a5e